### PR TITLE
88_Merge Sorted Array

### PR DESCRIPTION
**Pull request by djharshit**

### CODE REVIEW

**Comment by djharshit** [comment]
This PR has been excluded, it is saying this repository did not follow the hacktoberfest guidelines. Why ?

**Comment by ritikbanger** [comment]
> This PR has been excluded, it is saying this repository did not follow the hacktoberfest guidelines. Why ?

Because nobody is solving issues, everyone copy paste code for the t-shirt.

File: 04. Arrays/26_Remove Duplicates from Sorted Array.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Solution:
```
Comment (ritikbanger) [inline 04. Arrays/26_Remove Duplicates from Sorted Array.py:1]: Change file name, it should not contain any spaces, refer to readme.

File: 04. Arrays/88_Merge Sorted Array.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Solution:
```
Comment (ritikbanger) [inline 04. Arrays/88_Merge Sorted Array.py:1]: Change file name, it should not contain any spaces, refer to readme.

**Review by ritikbanger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**